<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentSystemClass</class>
 <widget class="QMainWindow" name="StudentSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StudentSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>681</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;学生信息管理系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QWidget" name="loginText" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>131</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>53</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;账号:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>53</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="AccountText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>151</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="adminPB">
        <property name="font">
         <font>
          <family>DengXian</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>管理员</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="userPB">
        <property name="font">
         <font>
          <family>DengXian</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="loginPB">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="registPB">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StudentSystem.qrc"/>
 </resources>
 <connections/>
</ui>
